app/src/main/java/com/pbm/Login.java

```diff
@@ -24,7 +24,7 @@
 import java.net.URLDecoder;
 import java.util.concurrent.ExecutionException;
 
-public class Login extends AppCompatActivity implements ActivityCompat.OnRequestPermissionsResultCallback{
+public class Login extends PinballMapActivity implements ActivityCompat.OnRequestPermissionsResultCallback{
```

Comment:
This is ALL I did in this commit. And the result is that if you try to login with an unconfirmed user, you get this error message. https://github.com/scottwainstock/pbm/commit/e69452690aaf72540663bbd818ea60a99dd7136e#diff-6a96117366c36e46f72e2f023808047eR30

Pretty cool!